apps/cli/test/credentialsStorage.test.ts

```diff
@@ -0,0 +1,71 @@
+import { fs, type DirectoryJSON, vol } from "memfs";
+import { beforeEach, describe, expect, it, vi } from "vitest";
+
+// Mock all fs promises functions
+vi.mock("node:fs/promises", () => fs.promises);
+
+// Use a mock homedir in order to reliably have a memfs home directory
+vi.mock("node:os", () => ({ homedir: () => "/home/codemod-test/" }));
```

Comment:
If there is a better way to enforce a homedirectory than by mocking an implementation internal, I'm down to do that instead.